packages/chart/src/ChartUtils.test.ts

```diff
@@ -557,22 +559,22 @@ it('creates range breaks from holidays correctly', () => {
 
 describe('axis property name', () => {
   it('gets x/y axis property names correctly', () => {
-    expect(ChartUtils.getAxisPropertyName(dh.plot.AxisType.X)).toBe('x');
-    expect(ChartUtils.getAxisPropertyName(dh.plot.AxisType.Y)).toBe('y');
+    expect(chartUtils.getAxisPropertyName(dh.plot.AxisType.X)).toBe('x');
+    expect(chartUtils.getAxisPropertyName(dh.plot.AxisType.Y)).toBe('y');
   });
 
   it('returns null for all other properties', () => {
-    expect(ChartUtils.getAxisPropertyName(dh.plot.AxisType.COLOR)).toBe(null);
-    expect(ChartUtils.getAxisPropertyName(dh.plot.AxisType.LABEL)).toBe(null);
-    expect(ChartUtils.getAxisPropertyName(dh.plot.AxisType.SHAPE)).toBe(null);
-    expect(ChartUtils.getAxisPropertyName(dh.plot.AxisType.SIZE)).toBe(null);
-    expect(ChartUtils.getAxisPropertyName(dh.plot.AxisType.Z)).toBe(null);
+    expect(chartUtils.getAxisPropertyName(dh.plot.AxisType.COLOR)).toBe(null);
+    expect(chartUtils.getAxisPropertyName(dh.plot.AxisType.LABEL)).toBe(null);
+    expect(chartUtils.getAxisPropertyName(dh.plot.AxisType.SHAPE)).toBe(null);
+    expect(chartUtils.getAxisPropertyName(dh.plot.AxisType.SIZE)).toBe(null);
+    expect(chartUtils.getAxisPropertyName(dh.plot.AxisType.Z)).toBe(null);
   });
 });
 
 describe('getPlotlyChartMode', () => {
   const { LINE, SCATTER } = dh.plot.SeriesPlotStyle;
-  const { getPlotlyChartMode } = ChartUtils;
+  const getPlotlyChartMode = chartUtils.getPlotlyChartMode.bind(chartUtils);
   test('scatter plots', () => {
```

Comment:
Side note, this would be a good example of using `test.each`, it would be something like:
```
test.each([
  [ SCATTER, null, null, 'markers' ],
  [ SCATTER, null, false, undefined ],
  ...
  [ LINE, true, true, 'lines+markers' ],
])('returns the expected plotly type for %s %s %s', (areLinesVisible, areShapesVisible, result) => {
  expect(getPlotlyChartMode(SCATTER, areLinesVisible, areShapesVisible)).toBe(result);
});
```

Reply:
Yep, much cleaner. Updated the tests.